pkg/acs/calacs/calacs/calacs.c

```diff
@@ -203,51 +206,49 @@ int CalAcsRun (char *input, int printtime, int save_tmp, int verbose, int debug,
         trlmessage("Finished MAMA processing...");
     }
 
-
-    /* Add DTH processing here... */
-    /* For each DTH product... */
+    /* CALACS no longer generates a drizzled product. Drizzle-combined data is generated   
+       by post-CALACS processing. Instead, the "ACSDTH" pipeline step can create both combined 
+       _spt.fits and .tra files. In order to have a combined output _spt.fits file generated, some 
+       _spt.fits files must exist for the filenames listed in the association table.
+    */
+ 
+    /* Loop over each entry in the association table to create a combined _spt.fits and a 
+       combined .tra file.  The terminology of "dth" is leftover from the original CALACS 
+       implementation.
+    */
     if (asn.process == FULL){
-        if (asn.verbose) {
-            trlmessage ("CALACS: Building DTH products");
```

Comment:
Should this really be removed? What if the output is parsed for this marker?

Reply:
Good point although CALACS has not actually built any true DTH products since 2002.  Even at that time the original _dth.fits products were actually dummy or empty files.  The changes made thus far are on the path to removing *any* indication of an ACSDTH "step" in CALACS after some discussion with the ACS team.   Any mention of an ACSDTH step has already been removed from the Data Handbook.